src/renderer/gdi/paint.cpp

```diff
@@ -592,6 +592,19 @@ using namespace Microsoft::Console::Render;
         cursorInvertRects.push_back(rcInvert);
         break;
 
+    case CursorType::DoubleUnderscore:
+    {
+        RECT top, bottom;
+        top = bottom = rcBoundaries;
+        RETURN_IF_FAILED(LongAdd(bottom.bottom, -1, &bottom.top));
+        RETURN_IF_FAILED(LongAdd(top.bottom, -3, &top.top));

```

Comment:
This one too might need to check DPIs. I know for single underline it's just -1 pixel off the bottom and it doesn't think about the DPI.... but my concern is that on a high DPI monitor that the two lines get so close to each other at only a 2px separation that they look like a weird fuzzy fat line instead of two lines.

Reply:
I can't tell. How do I check that? That is, how do I switch/change to the GDI renderer?

Reply:
This is tricky - technically, only conhost uses the GDI renderer, and conhost doesn't have a way of enabling this cursor style (nor do we plan on adding support for it to conhost). So this might _technically_ be better off with just no-op'ing here

Reply:
Hm, yeah, OK. I'm fine with that.